lua/wire/client/wire_expression2_editor.lua

```diff
@@ -1838,6 +1838,8 @@ function Editor:Close()
 	self:SetV(false)
 	self.chip = false
 
+	net.Start("IsEditing") net.WriteBool(false) net.SendToServer()
```

Comment:
'tis. This triggers on e2, gpu AND cpu. But it was only added for the latter two.